airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -237,6 +238,18 @@ def _apply_author_info_to_metadata_file(metadata_dict: dict, original_metadata_f
     return metadata_dict
 
 
+def _apply_sbom_url_to_metadata_file(metadata_dict: dict) -> dict:
+    """Apply sbom url to the metadata file before uploading it to GCS."""
+    try:
+        sbom_url = f"https://connectors.airbyte.com/files/sbom/{metadata_dict['data']['dockerRepository']}/{metadata_dict['data']['dockerImageTag']}.spdx.json"
+    except KeyError:
+        return metadata_dict
+    response = requests.head(sbom_url)
+    if response.ok:
+        metadata_dict = set_(metadata_dict, "data.generated.sbomUrl", sbom_url)
```

Comment:
For my education: do we put a lot of things in `generated` metadata key?

Reply:
This is what currently ends up as generated metadata. It's mainly to feed the doc with marketplace metrics.

<img width="749" alt="Screenshot 2024-08-20 at 09 52 46" src="https://github.com/user-attachments/assets/1d9109ee-c770-459d-9957-52b6b78b8bb4">